packages/ui-app/src/Input.tsx

```diff
@@ -172,6 +173,14 @@ export default class Input extends React.PureComponent<Props, State> {
     }
   }
 
+  private onBlur = (event: React.KeyboardEvent<Element>): void => {
+    const { onBlur } = this.props;
+
+    if (onBlur) {
+      onBlur(event);
```

Comment:
Not sure we need to pass the `event` back up - it makes sense for key-presses, but for the rest probably not.